auth/extensions.py

```diff
@@ -0,0 +1,26 @@
+from importlib import import_module
+from importlib.util import find_spec
+
+import logging
+
+from .credentials import allowed_extensions
+
+
+def action_user(extension, user_info={}):
+    if extension not in allowed_extensions:
```

Comment:
it's not "allowed_extensions" but "installed extensions" or "enabled extensions"